### PR TITLE
Add information on methods for mcse

### PR DESCRIPTION
**Pull request by rikhuijzer**

### CODE REVIEW

File: src/mcse.jl
Comment (devmotion) [inline src/mcse.jl:4]: Maybe keep
```suggestion
Compute the Monte Carlo standard error (MCSE) of samples `x`.
```
since it was supposed to improve the existing docstring in MCMCChains.

File: src/mcse.jl
Comment (devmotion) [inline src/mcse.jl:6]:
```suggestion

The optional argument `method` describes how the errors are estimated. Possible options are: 
```

File: src/mcse.jl
Comment (devmotion) [inline src/mcse.jl:9]: At some point we should explain it more clearly but this is definitely an improvement :slightly_smiling_face:

File: src/mcse.jl
Comment (devmotion) [inline src/mcse.jl:10]: It seems this line can be removed:
```suggestion
```

**Comment by rikhuijzer** [comment]
Thanks for the feedback, David. Let me know if I can also update the MCMCChains docs. I'm not sure what you want with that exactly. For me, closing that PR is also fine if you plan to merge the migration to InferenceDiagnostics.jl soon(ish).

**Comment by rikhuijzer** [comment]
According to Benjamin Deonovic on Slack. The methods are based on other calculations. Don't merge yet, I'll add refs.

**Comment by rikhuijzer** [comment]
Okay. Updated it. Are the refs okay like this? I see that you also use 

```
**references**

[...]
```
but that isn't so useful here. Alternatively, we can move the docs in the different methods and add a bunch of `[mcse_bm](@ref)` etc

**Comment by devmotion** [comment]
I think it's fine as it is. IIRC I did not use cross references since I was not satisfied with the Documenter output (e.g., there are issues with references that appear in different docstrings (https://github.com/JuliaDocs/Documenter.jl/issues/566) but maybe this is not an issue anymore since I removed the references from `FFTESSMethod`).

**Review by devmotion — APPROVED** [review]

**Comment by devmotion** [comment]
> Thanks for the feedback, David. Let me know if I can also update the MCMCChains docs. I'm not sure what you want with that exactly. For me, closing that PR is also fine if you plan to merge the migration to InferenceDiagnostics.jl soon(ish).

I plan to register the package within the next days (https://github.com/devmotion/InferenceDiagnostics.jl/issues/2#issuecomment-873638740) and there is already a PR for MCMCChains, so I assume the migration can be completed until the end of next week. So maybe we can just wait until end of next week and merge the PR to MCMCChains only if it is not fixed until then?

**Comment by rikhuijzer** [comment]
> > Thanks for the feedback, David. Let me know if I can also update the MCMCChains docs. I'm not sure what you want with that exactly. For me, closing that PR is also fine if you plan to merge the migration to InferenceDiagnostics.jl soon(ish).
> 
> I plan to register the package within the next days ([#2 (comment)](https://github.com/devmotion/InferenceDiagnostics.jl/issues/2#issuecomment-873638740)) and there is already a PR for MCMCChains, so I assume the migration can be completed until the end of next week. So maybe we can just wait until end of next week and merge the PR to MCMCChains only if it is not fixed until then?

Completely fine by me :) I'll just close the other PR for now.

**Comment by devmotion** [comment]
Thanks for the PR!

**Pull request merged**
